splinterd/man/splinterd.1.md

```diff
@@ -134,6 +134,9 @@ OPTIONS
   Any options on the command line will override the settings in the
   configuration file.
 
+`--display-name DISPLAY-NAME`
+: Specifies a human-readable name for the node (Default: "")
```

Comment:
The default value for display name was actually updated to `Node <node_id>`

Reply:
updated